opensrp-chw-hf/src/main/assets/rule/hiv_outcome_rules.yml

```diff
@@ -6,13 +6,27 @@ condition: "action_taken_hiv_problems.containsKey('tested')"
 actions:
   - "test_results_visibility =  true"
 ---
+name: "where_was_test_conducted_visibility"
+description: "where_was_test_conducted visibility"
+priority: 1
+condition: "action_taken_hiv_problems.containsKey('tested')"
+actions:
+  - "where_was_test_conducted_visibility =  true"
+---
 name: "reason_no_action_was_taken_visibility"
 description: "reason_no_action_was_taken visibility"
 priority: 1
 condition: "action_taken_hiv_problems.containsKey('no_action_taken')"
 actions:
   - "reason_no_action_was_taken_visibility =  true"
 ---
+name: "other_reason_no_action_was_taken_visibility"
+description: "other_reason_no_action_was_taken visibility"
+priority: 1
+condition: "reason_no_action_was_taken.containsKey('other') && action_taken_hiv_problems.containsKey('no_action_taken')"
+actions:
+  - "other_reason_no_action_was_taken_visibility =  true"
+---
```

Comment:
Same applies here, the added field should be other places where the test was conducted